isic/settings/development.py

```diff
@@ -37,6 +37,8 @@
 
 SECRET_KEY = "insecure-secret"
 
+RUNSERVERPLUS_SERVER_ADDRESS_PORT = env.str("DJANGO_RUNSERVERPLUS_SERVER_ADDRESS_PORT")
```

Comment:
```suggestion
RUNSERVERPLUS_SERVER_ADDRESS_PORT: str = env.str("DJANGO_RUNSERVERPLUS_SERVER_ADDRESS_PORT")
```

Django-environ lacks typing support, and I'm hoping having setting types will eventually be useful.